Makefile

```diff
@@ -73,7 +73,7 @@ docs: $(GO_SOURCES) ## Generate the plugin documentation
 
 .PHONY: test
 test: prepare## Run tests
-	go test ./... -coverprofile=coverage.txt -covermode=atomic -timeout 1m -race
+	go test ./... -coverprofile=coverage.txt -covermode=atomic -timeout 3m -race
```

Comment:
I see this change as part of #474 watch out for merge conflicts

Reply:
yes, will have it in mind